backend/.gitignore

```diff
@@ -1,3 +1,3 @@
 /node_modules
 .env
-/database/prisma/dev.db
\ No newline at end of file
+/database/prisma/dev.dbgit
```

Comment:
this got changed on accident it seems?

Reply:
i dont think this was me unless a misclick. dont remember changing this though